### PR TITLE
HTML/CSS Design Teardown

### PR DESCRIPTION
**Pull request by oloomoses**
We have created divs on the navigation bar to enable proper positioning of the individual elements

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @oloomoses and @gdumani 
### Good job you have done so far :clap: However, you have a few issues that you need to solve before your application can be approved.
- Be careful and strict with the number of columns you are required to put in your project, example if you have 3 columns put 3
3 columns
3 colums all
- Also, make the columns equal, the same space between them and they need to go full width.
- Remove commented code from  your html file
comment
### Make all these changes and resubmit for another code review. Good luck.
- Happy coding!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,416 @@
...
+    <script src="https://kit.fontawesome.com/a076d05399.js"></script>
+    <link rel="stylesheet" href="style.css"/>
+  </head>
+
+  <body>
+
+    <header>
+      <div class="logo"><h3>SMASHING</h3>MAGAZINE</div>
+     
+      <div class="nav">
```
Comment (euqueme) [inline index.html]: [OPTIONAL]

- [x] I strongly recommend changing this `<div>` tag to a `<nav>` tag because is more Semantic HTML coherent

**Review by euqueme — APPROVED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @oloomoses and @gdumani :balloon: :lollipop: 

### Thank you for doing the changes requested by the previous reviewer, :medal_sports: Your project looks amazing :sparkles:  since the previous reviewer didn't mention this I have to approve the changes but please consider doing this change before merging the branch:

## Building a Heat Map

[OPTIONAL]

- [ ] Heat maps should only use grayscale colors so please remove all images or change them to grayscale ones, and also blue links to a grayscale color to achieve what was asked in the project specs

image

### That's it, team! :raised_hands: 

### Please :pray:  do this change (or not) and merge this branch :memo:  because your project is approved! :heavy_check_mark: 

PROJECT STATUS: APPROVED :green_apple:

File: index.html (hunk 1 above)
Comment (gdumani) [reply index.html]: implemented optional recommendations

**Pull request merged**